ARE/middleware/src/main/java/eu/asterics/mw/are/AsapiSupport.java

```diff
@@ -216,39 +190,25 @@ public String getComponentDescriptorsAsXml() throws AREAsapiException {
 
                         @Override
                         public String call() throws Exception {
-                            logger.fine("\n\ngetComponentDescriptorsAsXml: \n\n");
+                            logger.fine("getComponentDescriptorsAsXml()");
                             String response = "";
-
-                            response += "<?xml version=\"1.0\"?>";
-                            response += "<componentTypes xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xsd=\"http://www.w3.org/2001/XMLSchema\">";
-
-                            List<Bundle> bundleList = DeploymentManager.instance.getBundleManager()
-                                    .getInstallableBundleList();
-                            for (Bundle bundle : bundleList) {
-                                URL bundleDescriptorURL = bundle
-                                        .getResource(DefaultBundleModelParser.BUNDLE_DESCRIPTOR_RELATIVE_URI);
-                                if (bundleDescriptorURL != null) {
-                                    try {
-                                        response += getFormattedBundleDescriptorStringOfComponentTypeId(
```

Comment:
why this formatting is not necessary anymore? (`getFormattedBundleDescriptorStringOfComponentTypeId()`)

Reply:
it has moved to https://github.com/asterics/AsTeRICS/blob/gen-component-collection-and-help/ARE/middleware/src/main/java/eu/asterics/mw/are/BundleManager.java#L747

Reply:
ah, ok. thx